community/sinit/files/sinit-launch-services.boot

```diff
@@ -3,18 +3,13 @@
 # for busybox
 sinit_run_getty() {
     for getty in 1 2 3 4 5 6; do
+	# for busybox
         respawn /sbin/getty 38400 tty${getty} 2>&1 &
+	# for ubase
+	# respawn /sbin/getty /dev/tty${getty} linux 2>&1 &
```

Comment:
Personally I prefer # comment after the fork '&' in this sittuation.

```
respawn /sbin/getty 38400 tty${getty} 2>&1 &         # busybox getty
￼# respawn /sbin/getty /dev/tty${getty} linux 2>&1 & # ubase getty
```

But @dylanaraps is the king of nitpicks so I wonder what he thinks?

@cemkeylan Can we use a different word than 'for' in  `# for busybox`

Because to me, all of the uses of the word "for" around the for loop makes it look confusing ;)

Also line 3 can be removed.

Thanks @cemkeylan